src/app/tabs/compile-tab.js

```diff
@@ -332,7 +332,7 @@ class CompileTab extends ViewPlugin {
               var result = yo`<div>${uploaded.map((value) => {
                 return yo`<div><b>${value.filename}</b> : <pre>${value.output.url}</pre></div>`
               })}</div>`
-              modalDialogCustom.alert(`Published ${contract.name} Metadata`, yo`<span>Metadata of ${contract.name.toLowerCase()} contract was published successfully.<br> <pre>${result}</pre> </span>`)
+              modalDialogCustom.alert(`Published "${contract.name}" Metadata`, yo`<span>Metadata of ${contract.name.toLowerCase()} was published successfully.<br> <pre>${result}</pre> </span>`)
```

Comment:
I'm sorry but could you please also add ""s to contract name in the text?
Metadata of "ballot" was.....